services/QuillLMS/client/app/bundles/Shared/utils/appSettingAPIs.ts

```diff
@@ -0,0 +1,22 @@
+import { mainApiFetch, requestFailed, handleRequestErrors } from './../../Staff/helpers/comprehension';
+
+export const handleHasAppSetting = async (hasAppSetting, setHasAppSetting, key: string) => {
```

Comment:
You'll want to be passing in a `setErrors` function here from the `Activities` component, otherwise these errors are just getting swallowed and won't be displayed anywhere if they exist